diff --git a/build.sbt b/build.sbt
index 475efd02..9a7cd226 100644
--- a/build.sbt
+++ b/build.sbt
@@ -144,29 +144,31 @@ val isTestJar = sys.props.getOrElse("jartest", "false").toBoolean
 
 val settingsToUse = if (isTestJar) { jartestSettings } else { defaultSettings }
 
+libraryDependencies ++= Seq(
+  // The single % is for java libraries
+  // the %% appends the version of scala used, and should be used for scala libraries;
+  // the %%% is for scala-js (and scala native).
+  "dev.zio"       %% "zio"                               % versions("zio"),
+  "dev.zio"       %% "zio-config"                        % versions("zio.config"),
+  "dev.zio"       %% "zio-config-magnolia"               % versions("zio.config"),
+  "dev.zio"       %% "zio-config-typesafe"               % versions("zio.config"),
+  "dev.zio"       %% "zio-logging"                       % versions("zio.logging"),
+  // This is needed because micrometer (see below) uses SLF4J
+  "dev.zio"       %% "zio-logging-slf4j-bridge"          % versions("zio.logging"),
+  "dev.zio"       %% "zio-metrics-connectors-micrometer" % versions("zio.metrics"),
+  "dev.zio"       %% "zio-streams"                       % versions("zio"),
+  "io.micrometer"  % "micrometer-registry-jmx"           % versions("jmx"),
+  "org.scala-lang" % "scala-reflect"                     % versions("reflect"),
+  "org.tinylog"    % "tinylog-api"                       % versions("tinylog"),
+  "org.tinylog"    % "tinylog-impl"                      % versions("tinylog"),
+  "dev.zio"       %% "zio-test"                          % versions("zio") % Test,
+  "dev.zio"       %% "zio-test-junit"                    % versions("zio") % Test,
+  "com.github.sbt" % "junit-interface"                   % "0.13.3"        % Test,
+  "junit"          % "junit"                             % "4.13.2"        % Test
+)
+
 lazy val commonSettings = Seq(
-  libraryDependencies ++= Seq(
-    // The single % is for java libraries
-    // the %% appends the version of scala used, and should be used for scala libraries;
-    // the %%% is for scala-js (and scala native).
-    "dev.zio"       %% "zio"                               % versions("zio"),
-    "dev.zio"       %% "zio-config"                        % versions("zio.config"),
-    "dev.zio"       %% "zio-config-magnolia"               % versions("zio.config"),
-    "dev.zio"       %% "zio-config-typesafe"               % versions("zio.config"),
-    "dev.zio"       %% "zio-logging"                       % versions("zio.logging"),
-    // This is needed because micrometer (see below) uses SLF4J
-    "dev.zio"       %% "zio-logging-slf4j-bridge"          % versions("zio.logging"),
-    "dev.zio"       %% "zio-metrics-connectors-micrometer" % versions("zio.metrics"),
-    "dev.zio"       %% "zio-streams"                       % versions("zio"),
-    "io.micrometer"  % "micrometer-registry-jmx"           % versions("jmx"),
-    "org.scala-lang" % "scala-reflect"                     % versions("reflect"),
-    "org.tinylog"    % "tinylog-api"                       % versions("tinylog"),
-    "org.tinylog"    % "tinylog-impl"                      % versions("tinylog"),
-    "dev.zio"       %% "zio-test"                          % versions("zio") % Test,
-    "dev.zio"       %% "zio-test-junit"                    % versions("zio") % Test,
-    "com.github.sbt" % "junit-interface"                   % "0.13.3"        % Test,
-    "junit"          % "junit"                             % "4.13.2"        % Test
-  ),
+  libraryDependencies := libraryDependencies.value,
   assembly / assemblyMergeStrategy := commonMergeStrategy,
   ThisBuild / assemblyShadeRules := shadeRules,
   assemblyPackageScala / assembleArtifact := false,
